contrib/cirrus/runner.sh

```diff
@@ -275,6 +275,7 @@ function _run_altbuild() {
         *Windows*)
             make podman-remote-release-windows_amd64.zip
             make podman.msi
```

Comment:
Huh. Should we replace this with building the new EXE installer? @n1hility

Reply:
I think #15610 is what does the EXE step, and if I read that correctly it can run only on Windows. This step here is a Linux one (the full name is "Windows Cross").

So maybe the question is, should we get rid of this step completely? Those Makefile steps are too convoluted for me to grok at a glance, but ISTR that they run a few doc checks that are important.